flocker/node/functional/test_deploy.py

```diff
@@ -24,6 +24,49 @@
 from ...route import make_memory_network
 
 
+class P2PNodeDeployer(object):
```

Comment:
Why not just update the tests that use this (now deleted) deployer? 
If only to limit required test changes and the size of the branch, then perhaps there should be a linked followup issue to eventually fix those tests.